oak/server/asylo/asylo_oak_manager.h

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+#ifndef ASYLO_OAK_MANAGER_H
```

Comment:
```suggestion
#ifndef OAK_SERVER_ASYLO_ASYLO_OAK_MANAGER_H_
```
as per https://google.github.io/styleguide/cppguide.html#The__define_Guard